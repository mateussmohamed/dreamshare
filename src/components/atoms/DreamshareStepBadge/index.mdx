---
name: DreamshareStepBadge
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import DreamshareStepBadge from './index'

# DreamshareStepBadge
<PropsTable of={DreamshareStepBadge} />

<Playground>
  <DreamshareStepBadge>
    Step 1
  </DreamshareStepBadge>
</Playground>
