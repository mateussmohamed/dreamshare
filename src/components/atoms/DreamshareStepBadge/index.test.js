import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import DreamshareStepBadge from './index'

describe('[<DreamshareStepBadge />]', () => {
  const wrap = (props = {}) =>
    shallow(<DreamshareStepBadge {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am badge' })

    expect(wrapper).toMatchSnapshot()
  })
})
