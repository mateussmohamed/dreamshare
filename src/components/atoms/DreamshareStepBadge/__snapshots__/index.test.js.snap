// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<DreamshareStepBadge />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "DreamshareStepBadge-sc-1mswk3p-0",
        "isStatic": false,
        "rules": Array [
          "padding:3px 8px;border-radius:20px;background-color:",
          [Function],
          ";",
          [Function],
          ";font-size:12px;letter-spacing:2px;text-align:center;text-transform:uppercase;color:#fff;",
        ],
      },
      "displayName": "DreamshareStepBadge",
      "render": [Function],
      "styledComponentId": "DreamshareStepBadge-sc-1mswk3p-0",
      "target": "span",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am badge
</StyledComponent>
`;
